### DISCUSSION TITLE
Install Error: so-yara-download failed

### DISCUSSION THREAD

**Discussion by EddieN17** [Security-Onion-Solutions/securityonion#10879] [2.4]
Hello,

Running SO 2.4.4 beta running the ISO image on a VMWare ESXi VM. I was installing the manager node and got an error after install. My setup has internet access but has to go through a proxy which appears to be the problem with the error. The error is:
```
[ERROR   ] Command '/usr/sbin/so-yara-download' failed with return code: 2
[ERROR   ] stderr: /usr/sbin/so-yara-download: line 8: syntax error near unexpected token `('
[ERROR   ] retcode: 2
[ERROR   ] {'pid': 40845, 'retcode': 2, 'stdout': '', 'stderr': "/usr/sbin/so-yara-download: line 8: syntax error near unexpected token `('\n/usr/sbin/so-yara-download: line 8: `export no_proxy= salt['pillar.get']('manager:no_proxy')'"}
      Result: False
```

Looking at the script in /usr/sbin/so-yara-download and /sbin/so-yara-download, the http_proxy and https_proxy was called for successfully but the no_proxy failed. With my webproxy configuration settings showing in the script.
(Snippet from script)
```
export http_proxy=http://webproxy.redacted:8080
export https_proxy=http://webproxy.redacted:8080
export no_proxy= salt['pillar.get']('manager:no_proxy')
```
 
I modified the script and set `no_proxy=$(salt-call pillar.get manager:no_proxy)` and after that ran `so-yara-download` and all worked well with that. But it would be nice if this script could be fixed through install.

Thanks for all the work with 2.4!

**Comment by dougburks** [comment]
We've created an issue for this and it should be fixed in the next release:
https://github.com/Security-Onion-Solutions/securityonion/issues/10880
